apps/rush/src/RushVersionManager.ts

```diff
@@ -15,6 +15,11 @@ import RushWrapper from './RushWrapper';
  *  installation and calling convention.
  */
 const RUSH_TRANSITIONAL_VERSION: string = '4.0.0';
+
+/**
+ * lib/rush.js was renamed to start.js just for rush 3.0.20.
```

Comment:
>lib/rush.js was renamed to start.js just for rush 3.0.20. [](start = 3, length = 57)

I interpreted this comment to mean that it's start.js for versions >= 3.0.20

Could you clarify that start.js was just this one release?

Reply:
I'm not sure how it could be more clear than "just for rush 3.0.20," but I can add another sentence that says it was rush.js for other versions.

---
In reply to: [148646058](https://github.com/Microsoft/web-build-tools/pull/428#discussion_r148646058) [](ancestors = 148646058)